### DISCUSSION TITLE
Skins not Applying to Debuffs in VuhDo

### DISCUSSION THREAD

**Discussion by Hugeplays** [SFX-WoW/Masque#213] [Issues] [answered]
### Game Flavor

Retail

### Game Version

9.0.1

### Add-On Version
Masque-9.0.1, VuhDo-v3.115

### Describe the Issue

Masque does not apply skins to any Debuffs in VuhDo. The skins are correctly applied to other elements in VuhDo however, e.g. HoT Icons.

### Screenshots
![bug](https://user-images.githubusercontent.com/73003874/96322829-8c0c3c80-1012-11eb-9487-aa495ebd4cd7.jpg)

### Notes

This bug is present without any other addons installed and has been recreated by several other players who are experiencing the same issue.

**Comment by StormFX** [comment, marked as answer]
I looked at VuhDo's code and I'm not seeing where it's adding debuffs to Masque. As far as I can tell, it only adds HoTs and buffs. This is mentioned [here](https://gitlab.vuhdo.io/vuhdo/vuhdo/-/issues/176), as well. To be clear, Masque only skins buttons/icons that an add-on tells it to skin.

**Comment by Hugeplays** [comment]
That makes sense; would you recommend that I get in touch with the author of VuhDo instead then?

**Comment by StormFX** [comment]
That would be the best bet. Keep in mind that there's already an existing issue for this, as I linked above.
